src/host/ncp_spinel.cpp

```diff
@@ -741,6 +748,12 @@ otbrError NcpSpinel::HandleResponseForPropSet(spinel_tid_t      aTid,
         otbrLogInfo("Infra If handle ICMP6 ND result: %s", spinel_status_to_cstr(status));
         break;
 
+    case SPINEL_PROP_DNSSD_STATE:
+        VerifyOrExit(aKey == SPINEL_PROP_LAST_STATUS, error = OTBR_ERROR_INVALID_STATE);
+        SuccessOrExit(error = SpinelDataUnpack(aData, aLength, SPINEL_DATATYPE_UINT_PACKED_S, &status));
+        otbrLogInfo("Update Dnssd state result: %s", spinel_status_to_cstr(status));
```

Comment:
nit: how about just making it all lower case?
```suggestion
        otbrLogInfo("Update dnssd state result: %s", spinel_status_to_cstr(status));
```

Reply:
Updated!